TheMovieDB/TheMovieDB/Presentation/Coordinator/Storyboeared.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Storyboeared.swift
+//  TheMovieDB
+//
+//  Created by Agustin Godoy Cosser on 12/10/2022.
+//
+
+import Foundation
+import UIKit
+
+protocol Storyboarded {
+    static func instantiate() -> Self?
+}
```

Comment:
should be inside a protocol folder

Reply:
https://github.com/agustingodoyc/TheMovieDB/pull/2/commits/306178015c7980b8d5a5be341797b3c21ffb3e8d